app/schema/widgets/multiple_choice_widget.py

```diff
@@ -62,8 +62,9 @@ def find_other_value(posted_data, options):
         Compare the posted_data with the options in the schema, if there is a value which doesn't match
         the options it must be the other value
         """
-        if posted_data and 'other' in posted_data:
-            for answer in posted_data:
-                if answer and not answer.isspace() and not any(option['value'] == answer for option in options) and answer is not 'other':
+        if posted_data and 'other' in (value.lower() for value in posted_data):
+            for answer in filter(None, [value.strip() for value in posted_data]):
```

Comment:
The other way doesn't get rid of the spaces. Maybe that's valid, but even if not it should arguably be done in `get_other_value` in the widget instead. I'll put this back the way it was for now.